event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/EventStreamRepository.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.services.eventsourcing.repository.jdbc;
+
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStreamObject;
+
+import java.util.UUID;
+import java.util.stream.Stream;
+
+public interface EventStreamRepository {
+
+    Stream<EventStreamObject> getBySequenceId(final Long sequenceNumber);
+
+    void markActive(final UUID streamId, final boolean active);
+
+    long getCurrentSequence();
+
+    void store(final UUID streamId);
```

Comment:
So we store an streamId but return an EventStreamObject - is that clear/consistent?

Reply:
Your right, Changed to createEventStreamObject instead of Store